src/main/features/response/routes/statement-of-means/monthly-income.ts

```diff
@@ -13,21 +13,64 @@ import { User } from 'idam/user'
 import { RoutablePath } from 'shared/router/routablePath'
 import { ResponseDraft } from 'response/draft/responseDraft'
 import { Draft } from '@hmcts/draft-store-client'
+import { CalculateMonthlyIncomeExpense } from 'common/calculate-monthly-income-expense/calculateMonthlyIncomeExpense'
+import { IncomeSource } from 'response/form/models/statement-of-means/incomeSource'
+import { IncomeExpenseSources } from 'common/calculate-monthly-income-expense/incomeExpenseSources'
+import { Validator } from 'class-validator'
 
 const page: RoutablePath = StatementOfMeansPaths.monthlyIncomePage
 
 function renderView (form: Form<MonthlyIncome>, res: express.Response): void {
   res.render(page.associatedView, {
-    form: form
+    form: form,
+    totalMonthlyIncomeExpense: calculateTotalMonthlyIncomeExpense(form.model)
   })
 }
 
+function calculateTotalMonthlyIncomeExpense (model: MonthlyIncome): number {
+  if (!model) {
+    return undefined
+  }
+  const incomeExpenseSources = IncomeExpenseSources.fromFormModel(model)
+
+  if (!isValid(incomeExpenseSources)) {
+    return undefined
+  }
+
+  return CalculateMonthlyIncomeExpense.calculateTotalAmount(
+    incomeExpenseSources.incomeExpenseSources
+  )
+}
+
+function isValid (incomeExpenseSources: IncomeExpenseSources): boolean {
+  const validator = new Validator()
+  const errors = validator.validateSync(incomeExpenseSources)
```

Comment:
I think this is simpler:
`return validator.validateSync(incomeExpenseSources).length === 0`